src/schema/resolvers.ts

```diff
@@ -21,8 +21,8 @@ export const resolvers = {
     },
     users: async (_: any, { data: args }: { data: UsersInput }, context: any): Promise<UsersType> => {
       verifyAuthOrFail(context);
-      const skip = Number(args?.skip) ?? 0;
-      const take = Number(args?.take) ?? 10;
+      const skip = +(args?.skip ?? 0);
+      const take = +(args?.take ?? 10);
```

Comment:
```suggestion
      const skip = +args?.skip ?? 0;
      const take = +args?.take ?? 10;
```

Não precisa transformar o 0 e o 10 🤣

Reply:
Precisa porque o TS reclama dizendo que `args?.skip` por ser `undefined` :/
<img width="566" alt="Screen Shot 2021-05-12 at 10 00 27" src="https://user-images.githubusercontent.com/46571639/117979045-e2bbc980-b308-11eb-91b1-dee870ddda76.png">

Reply:
Ah, maldito strict.

Reply:
Mas pera, o skip já viria como number, não precisa transformar né?

Reply:
Ta certissimo. Vou tirar o `+`